README.md

```diff
@@ -48,159 +47,156 @@ limitations under the License.
+## Install 💽
+Tested on Python 3.6+, and ONNX 1.5.0+. [virtual environment](https://docs.python.org/3/library/venv.html) is highly recommended!!
 
-<img src="https://docs.neuralmagic.com/docs/source/infographics/deepsparse.png" width="960px" />
+```bash
+pip install deepsparse
+```
+__ __
+## Quick Start 💻
 
-## Highlights
+One of the greatest features of our inference engine is the ability to integrate into popular deep learning libraries (e.g. Hugging Face, Ultralytics) allowing you to leverage DeepSparse for loading and deploying sparse models with ONNX runtime (ORT). ORT gives flexibility to serve your model in a framework agnostic environment. Support includes [PyTorch,](https://pytorch.org/docs/stable/onnx.html) [TensorFlow,](https://github.com/onnx/tensorflow-onnx) [Keras,](https://github.com/onnx/keras-onnx) and [many other frameworks](https://github.com/onnx/onnxmltools).
 
-<p>
-    <a href="https://neuralmagic.com/blog/benchmark-resnet50-with-deepsparse/">
-        <img alt="ResNet-50, b64 - ORT: 296 images/sec vs DeepSparse: 2305 images/sec on 24 cores" src="https://docs.neuralmagic.com/docs/source/highlights/deepsparse/resnet-50.png" width="256px" />
-    </a>
-    <a href="https://neuralmagic.com/blog/benchmark-yolov3-on-cpus-with-deepsparse/">
-        <img alt="YOLOv3, b64 - PyTorch: 6.9 images/sec vs. DeepSparse: 46.5 images/sec" src="https://docs.neuralmagic.com/docs/source/highlights/deepsparse/yolov3.png" width="256px" />
-    </a>
-</p>
+To begin your inference adventure with DeepSparse is super simple. You can either run DeepSparse in CLI or in Python. Let's first check out the CLI features... 👇
 
+## DeepSparse Server & Benchmark in CLI
 
-## Tutorials
+After you've installed DeepSparse in your Python environment, you have two resources via CLI at your disposal:
 
-- [Benchmarking with deepsparse.benchmark](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/benchmark_model)
-- [Image Classification](https://github.com/neuralmagic/deepsparse/tree/main/examples/classification)
-- [Object Detection](https://github.com/neuralmagic/deepsparse/tree/main/examples/detection)
-- [Flask Serving](https://github.com/neuralmagic/deepsparse/tree/main/examples/flask)
-- [YOLOv3](https://github.com/neuralmagic/deepsparse/tree/main/examples/ultralytics-yolov3)
+`deepsparse.server` : If you are interested in serving your models.
 
-## Installation
+`deepsparse.benchmark` : If you are interested in experimenting with the throughput/latency performance of your models under different inference scenarios.
 
-This repository is tested on Python 3.6+, and ONNX 1.5.0+. It is recommended to install in a [virtual environment](https://docs.python.org/3/library/venv.html) to keep your system in order.
+### 1. DeepSparse Server
 
-Install with pip using:
+The DeepSparse inference server allows you to serve models and pipelines in deployment. The server runs on top of the popular FastAPI web framework and Uvicorn web server.
```

Comment:
```suggestion
The DeepSparse inference server allows you to serve models and pipelines in deployment. The server runs on top of the popular FastAPI web framework and Unicorn web server.
```

Reply:
it's Uvicorn :) https://www.uvicorn.org/